### PR TITLE
ODNI fas arab latn 2015

### PR DESCRIPTION
**Pull request by AhMohsen46**

### CODE REVIEW

File: maps/odni-fas-Arab-Latn-2015.yaml [hunk 1]
```diff
@@ -0,0 +1,393 @@
+---
+authority_id: odni
+id: 2015
+language: ics-630-01:prs
```
Comment (ronaldtse) [inline maps/odni-fas-Arab-Latn-2015.yaml]: `language: iso-639-2:fas`

File: maps/odni-fas-Arab-Latn-2015.yaml [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+  within the context of a report, where
+  additional information about the individual is provided,
+  the referent will be clearly identified.
+  This system will be used in conjunction with online tools,
+  name dictionaries, and lists containing
+  conventional spellings of names of well-known individuals.
+
+notes:
+  - |
+    - Alef maqsura (final yeh pronounced as a long “a”) should
```
Comment (ronaldtse) [inline maps/odni-fas-Arab-Latn-2015.yaml]: Can we split these into multiple notes?

File: maps/odni-per-Arab-Latn-2015.yaml [hunk 3]
```diff
@@ -0,0 +1,217 @@
+---
+authority_id: odni
+id: 2015
+language: ics-630-01:prs
+source_script: Afghan (Dari)
```
Comment (ronaldtse) [inline maps/odni-per-Arab-Latn-2015.yaml]: This has to be a 15924 code, so it would be `Arab`.

File: maps/odni-fas-Arab-Latn-2015.yaml [hunk 4]
```diff
@@ -0,0 +1,393 @@
+---
+authority_id: odni
+id: 2015
+language: ics-630-01:prs
+source_script: Persian (Farsi)
```
Comment (ronaldtse) [inline maps/odni-fas-Arab-Latn-2015.yaml]: This is still `Arab` even though it feels wrong...

File: maps/odni-per-Arab-Latn-2015.yaml [hunk 5]
```diff
@@ -0,0 +1,217 @@
+---
+authority_id: odni
+id: 2015
+language: ics-630-01:prs
```
Comment (ronaldtse) [inline maps/odni-per-Arab-Latn-2015.yaml]: `language: iso-639-3:prs`

File: maps/odni-per-Arab-Latn-2015.yaml [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+  form will occur without the use of diacritics. However,
+  within the context of a report, where
+  additional information about the individual is provided,
+  the referent will be clearly identified.
+  This system will be used in conjunction with online tools,
+  name dictionaries, and lists containing
+  conventional spellings of names of well-known individuals.
+
+notes:
+  - |
```
Comment (ronaldtse) [inline maps/odni-per-Arab-Latn-2015.yaml:78]: Please split into multiple notes.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: maps/odni-fas-Arab-Latn-2015.yaml (hunk 2 above)
Comment (AhMohsen46) [reply maps/odni-fas-Arab-Latn-2015.yaml]: yes sure, will do that now

File: maps/odni-per-Arab-Latn-2015.yaml (hunk 3 above)
Comment (AhMohsen46) [reply maps/odni-per-Arab-Latn-2015.yaml]: @ronaldtse 
so it'd be like this
source_script: Arab
?

**Comment by ronaldtse** [comment]
Thank you @AhMohsen46 !

**Pull request merged**